docs/proposals/odo-debug.md

```diff
@@ -0,0 +1,82 @@
+# odo debug support proposal
+
+Table of Contents:
+- [odo debug support proposal](#odo-debug-support-proposal)
```

Comment:
Not sure what you mean :-( 
Something like general reasoning why we need this command?

Reply:
Yes, something like what it's meant to do and what it's not meant to do.

Reply:
Allow remote debugging of the java and nodejs components.

^ Something like this?

Reply:
I was suggesting more along the lines of clarifying that `odo debug` is meant for debugging the components being deployed through `odo` and not debugging `odo` itself. For the latter, running `odo` with `-v` flag is the right choice. I was suggesting this because `debug` and `verbose` are often used interchangeably in practice and it could cause confusion to the end user.

> Allow remote debugging of the java and nodejs components.

Yeah. And a list of components that have debug support and are planned to be added, maybe. 

Does that make sense?